Add missing hungarian translations.

albertborsos opened the pull request:
<!--
Note that only PHP 7 and PHP 8 compatibility fixes are accepted. Please report security issues to maintainers privately.
-->

| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌
| Tests pass?   | ✔️

marcovtwout reviewed (approved):
Hard for me to review fully, but I assume you took the array keys from the latest english file?
Could you also add a changelog entry?

albertborsos commented:
I executed the `./yiic message messages/config.php` command from the framework folder which updated the current translation file. Then I removed the  translations marked with `@@`s and translated the new ones to hungarian.

Then I executed the `./yiic message messages/config.php` command again, which sorted the keys.

I added this modification to the changelog.

Thanks
